.github/workflows/catkin_test_build.yaml

```diff
@@ -0,0 +1,43 @@
+name: ROS1 test build
+
+on:
+  push:
+    branches:
+      - 'ros1-devel'
```

Comment:
Probably it would be better to run it on branch `ros1` and on PRs

Reply:
Done